apps/emqx_modules/src/emqx_delayed_api.erl

```diff
@@ -160,24 +160,20 @@ schema("/mqtt/delayed/messages") ->
 
 fields("message_without_payload") ->
     [
-        {msgid, mk(integer(), #{desc => <<"Message Id (MQTT message id hash)">>})},
-        {node, mk(binary(), #{desc => <<"The node where message from">>})},
-        {publish_at, mk(binary(), #{desc => <<"Client publish message time, rfc 3339">>})},
-        {delayed_interval, mk(pos_integer(), #{desc => <<"Delayed interval, second">>})},
-        {delayed_remaining, mk(non_neg_integer(), #{desc => <<"Delayed remaining, second">>})},
-        {expected_at, mk(binary(), #{desc => <<"Expect publish time, rfc 3339">>})},
-        {topic, mk(binary(), #{desc => <<"Topic">>, example => <<"/sys/#">>})},
-        {qos, mk(emqx_schema:qos(), #{desc => <<"QoS">>})},
-        {from_clientid, mk(binary(), #{desc => <<"From ClientId">>})},
-        {from_username, mk(binary(), #{desc => <<"From Username">>})}
+        {msgid, mk(integer(), #{desc => ?DESC(msgid)})},
+        {node, mk(binary(), #{desc => ?DESC(node)})},
+        {publish_at, mk(binary(), #{desc => ?DESC(publish_at)})},
+        {delayed_interval, mk(pos_integer(), #{desc => ?DESC(delayed_interval)})},
+        {delayed_remaining, mk(non_neg_integer(), #{desc => ?DESC(delayed_remaining)})},
+        {expected_at, mk(binary(), #{desc => ?DESC(expected_at)})},
+        {topic, mk(binary(), #{desc => ?DESC(topic), example => <<"/sys/#">>})},
+        {qos, mk(emqx_schema:qos(), #{desc => ?DESC(qos)})},
+        {from_clientid, mk(binary(), #{desc => ?DESC(from_clientid)})},
+        {from_username, mk(binary(), #{desc => ?DESC(from_username)})}
```

Comment:
Is there any way to avoid `{field, mk(desc => ?DESC(field)}` construct? Hocon could derive it implicitly, perhaps.

Reply:
Yes, I think so, the content in ?DESC() is an ID, most times it can be the same as the field name